x/ibc/04-channel/types/msgs.go

```diff
@@ -489,5 +489,5 @@ func (msg MsgAcknowledgement) GetSigners() []sdk.AccAddress {
 
 // Type implements sdk.Msg
 func (msg MsgAcknowledgement) Type() string {
-	return "ics04/opaque"
+	return "ics04/acknowledgement"
```

Comment:
same here, maybe `acknowledge_packet`?
```suggestion
	return "acknowledge_packet"
```